src/engraving/libmscore/excerpt.cpp

```diff
@@ -1391,6 +1391,9 @@ void Excerpt::cloneStaff2(Staff* srcStaff, Staff* dstStaff, const Fraction& star
 
     for (Measure* m = m1; m && (m != m2); m = m->nextMeasure()) {
         Measure* nm = score->tick2measure(m->tick());
+        if (!nm) {
```

Comment:
Could you please explain why it is null? I don't really get it from the commit description